res/css/views/settings/tabs/user/_KeybindingsUserSettingsTab.scss

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2021 Šimon Brandner <simon.bra.ag@gmail.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_KeybindingUserSettingsTab {
+    .mx_SettingsTab_section {
+        margin-bottom: 30px;
+    }
+
+    .mx_KeybindingUserSettingsTab_keybind {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        padding-bottom: 3px;
+
+
+        .mx_KeybindingUserSettingsTab_keybind_buttons {
+            display: flex;
+            align-items: center;
+
+            :nth-child(n + 1) {
+                margin-inline-end: 5px;
```

Comment:
Since `mx_KeybindingUserSettingsTab_keybind` is already a flex container you could move that out of here and use the CSS property [`gap`](https://developer.mozilla.org/en-US/docs/Web/CSS/gap#flex_layout) that now also applies to flex layout

Reply:
/me remembers the moment when he copied this code and said to himself _I will think about this later_ :D